packages/battery/battery_linux/LICENSE

```diff
@@ -0,0 +1,27 @@
+// Copyright 2020 The Chromium Authors. All rights reserved.
```

Comment:
The Flutter Authors

This applies throughout the PR. (The current files are inconsistent, which I'm working on fixing, but there's no reason for new files to use legacy author lines.)

Reply:
Done.